dkan.profile

```diff
@@ -501,7 +502,9 @@ edit-menu-description
     ))
     ->condition('eid', '5')
     ->execute();
-  drupal_flush_all_caches();
+}
+
+function dkan_post_install() {
```

Comment:
@topicus can you explain what you're doing here? I don't understand this change or the idea behind the dkan_post_install() function